config/installation_config.yml

```diff
@@ -41,67 +56,138 @@
   display_title: 'Chatwoot Metadata'
   description: 'Display default Chatwoot metadata like favicons and upgrade warnings'
   type: boolean
-- name: MAILER_INBOUND_EMAIL_DOMAIN
-  value:
-  locked: false
-- name: MAILER_SUPPORT_EMAIL
-  value:
+# ------- End of Branding Related Config ------- #
+
+
+
+# ------- Signup & Account Related Config ------- #
+- name: ENABLE_ACCOUNT_SIGNUP
+  display_title: 'Enable Account Signup'
+  type: boolean
+  description: 'Allow users to signup for new accounts'
+  value: false
+  description: 'Allow users to signup for new accounts'
   locked: false
 - name: CREATE_NEW_ACCOUNT_FROM_DASHBOARD
   value: false
+  description: 'Allow users to create new accounts from the dashboard'
   locked: false
-- name: INSTALLATION_EVENTS_WEBHOOK_URL
+- name: HCAPTCHA_SITE_KEY
   value:
   locked: false
-- name: CHATWOOT_INBOX_TOKEN
+- name: HCAPTCHA_SERVER_KEY
   value:
   locked: false
-- name: CHATWOOT_INBOX_HMAC_KEY
+- name: INSTALLATION_EVENTS_WEBHOOK_URL
   value:
+  display_title: 'System events Webhook URL'
+  description: 'The URL to which the system events like new accounts created will be sent'
   locked: false
-- name: API_CHANNEL_NAME
-  value:
-- name: API_CHANNEL_THUMBNAIL
-  value:
-- name: ANALYTICS_TOKEN
-  value:
 - name: DIRECT_UPLOADS_ENABLED
+  type: boolean
   value: false
+  description: 'Enable direct uploads to cloud storage'
   locked: false
-- name: HCAPTCHA_SITE_KEY
+# ------- End of Account Related Config ------- #
+
+
+
+# ------- Email Related Config ------- #
+- name: MAILER_INBOUND_EMAIL_DOMAIN
   value:
+  description: 'The domain name to be used for generating conversation continuity emails (reply+id@domain.com)'
   locked: false
-- name: HCAPTCHA_SERVER_KEY
+- name: MAILER_SUPPORT_EMAIL
   value:
   locked: false
-- name: LOGOUT_REDIRECT_LINK
-  value: /app/login
+# ------- End of Email Related Config ------- #
+
+
+# ------- Facebook Channel Related Config ------- #
+- name: FB_APP_ID 
+  display_title: 'Facebook App ID'
   locked: false
-- name: DISABLE_USER_PROFILE_UPDATE
-  value: false
+- name: FB_VERIFY_TOKEN 
+  display_title: 'Facebook Verify Token'
```

Comment:
```suggestion
  display_title: 'Facebook webhook verification token'
```

Reply:
long name will break the UI. Added this as description instead